Anguandia/quickcredit
# Add get all users feature

#### What does this pull request do?

Add get-all-users feature

#### Description of Task to be completed?

- Add functionality to the user list placeholder controller function

#### How should this be manually tested?

- On postman, sign up on post/api/v1/auth/signup with your names, email and isAdmin as true
- Copy the token in the response and update the auth token on postman
- Make a GET/api/v1/users request to view all users
- you should see an empty array

#### What are the relevant pivotal tracker stories?

[finishes #166082459]
